.github/ISSUE_TEMPLATE.md

```diff
@@ -43,4 +43,3 @@ https://stackoverflow.com/questions/tagged/cheroot+or+cherrypy
 - Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
 
 📋 **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, e.g. stackoverflow, gitter, etc.)
-
```

Comment:
I think this change belongs to a separate PR. We value atomic changes (for both PRs and individual commits) as it makes Git history clean and better usable.

Here's some articles explaining best ways to write commit messages:
  http://chris.beams.io/posts/git-commit/
  http://alistapart.com/article/the-art-of-the-commit
  https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message
  https://github.com/erlang/otp/wiki/writing-good-commit-messages
  https://wiki.openstack.org/wiki/GitCommitMessages
  http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
  https://dev.to/gonedark/when-to-make-a-git-commit

Reply:
That got placed in there by a modification by tox. Albeit, I've not worked with the framework very much so I had assumed it wouldn't modify my files unless I told it to. I would agree, moving that elsewhere would be beneficial.

Reply:
Ah, it's probably done by `pre-commit` tool :) One of the hooks does it.